care/facility/api/serializers/patient_external_test.py

```diff
@@ -117,6 +131,14 @@ class Meta:
         model = PatientExternalTest
         fields = ("address", "ward", "local_body", "patient_created")
 
+    def validate(self, attrs):
+        user = self.context["request"].user
+        if user.user_type < User.TYPE_VALUE_MAP["Nurse"]:
+            raise ValidationError(
+                {"user": ["User is not allowed to perform this action"]}
+            )
+        return super().validate(attrs)
```

Comment:
Same, Why are Authz checks done in the serializer validate? Shouldn't this be done by DRY Permissions?